src/plus/webviews/graph/graphWebview.ts

```diff
@@ -539,6 +543,16 @@ export class GraphWebview extends WebviewBase<State> {
 		this.updateColumn(e.name, e.config);
 	}
 
+	private onSwitchToRef(e: SwitchToRefParams) {
+		const item: GraphItemContext = JSON.parse((e.ref as any).context);
```

Comment:
@eamodio @d13 I'm not sure if it is the best way of doing that (using the `context` string property). 

What do you guys think?